### ISSUE TITLE
Question: Create file object from path?

### ISSUE THREAD

**Issue by martin-john-green** [alphaleonis/AlphaFS#153]
Is there a simple way of constructing the type of file object returned by (say) Get-ChildItem (PowerShell) from the long pathnames returned by AlphaFS?

**Comment by Yomodo** [comment]
Perhaps our Wiki can help you?
https://github.com/alphaleonis/AlphaFS/wiki/PowerShell

**Comment by Yomodo** [comment]
@martin-john-green Were you able to find what you need?

**Comment by martin-john-green** [comment]
Thanks for your help.

I realised that could get what I needed from the attributes of the file, rather than the properties of an object representing the file. 

I found it easier to get the file attributes as a bitmap then mask them, rather than using FileInfo and FileSystemEntryInfo (I am using the AlphaFS classes to do this) 

I am currently recoding some inherited PowerShell code to remove use of relative paths (small changes) and to use functions that don’t break when the path gets too long (more substantial changes)

I am also advising other people at work to use AlphaFS!
